src/library/Box/propTypes.js

```diff
@@ -29,5 +31,7 @@ export const boxPropTypes = {
   padddingStart: spacingPropType,
   paddingTop: spacingPropType,
   paddingVertical: spacingPropType,
+  position: string,
+  scrollable: bool,
```

Comment:
`scrollable` is a shorthand convenience. We need to also support passthrough of `overflow` as well.